js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,34 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="pattern"
+        labelIcon={
+          <HelpItem
+            helpText={t("targetContextAttributesHelp")}
+            fieldLabelId="clients:targetContextAttributes"
+          />
+        }
+        isRequired
+        validated={errors.pattern ? "error" : "default"}
+        helperTextInvalid={t("required")}
+      >
+        <Controller
+          name="targetContextAttributes"
+          defaultValue={false}
+          control={control}
+          render={({ field }) => (
+            <Checkbox
+              id="required"
+              data-testid="standard"
+              name="required"
```

Comment:
There is no need to specify a `name` for this field, this is already handled by React Form Hook.

Reply:
correct! the chackbox's name has been removed